docs/openapi/api-eservice-operations-interop-v1.yaml

```diff
@@ -435,16 +475,36 @@ components:
           title: total elements
           type: integer
           format: int64
-    EserviceState:
+          
+    SearchProducerNameResponse:
+      type: object
+      properties:
+        label:
+          title: label of producer's name
+          type: string
+        value:
```

Comment:
yes, but the FE component retrieves a 'label,value' object and since we do not have a label but only the value we insert the same content in both attributes

Reply:
we can include it as tech debt for the next sprint'

Reply:
ok